docker-compose.yml

```diff
@@ -17,14 +15,13 @@ services:
     user: "101:101" # uid/gid used by nginx image
     volumes:
       - ./www:/var/www/html
-      - ./logs:/usr/local/log
     restart: unless-stopped
   nginx:
-    image: nginx:1.20.1
+    image: nginx:1.26.3
     volumes:
       - ./www:/var/www/html
       - ./logs:/var/log/nginx
-      - ./nginx/conf.d:/etc/nginx/conf.d
+      - ./nginx/conf.d:/etc/nginx/conf.d:ro
       - ./nginx/includes:/etc/nginx/includes
```

Comment:
Just noting I left this one as `rw` due to LE certs being in this folder on `osumirror.idle.host`

Reply:
I think it should still work, as long as you're doing that from the host or another container? The `ro` here should only apply to the `bind` mount on this container's side.

Reply:
You're correct, it'll be fine as LE is renewing from the host instead of container.